File: helloserver.go
package main

import (
	"log"
	"net/http"
)

func main() {
	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
		w.Write([]byte("Hello, " + r.URL.Path))
	})

	address := ":9991"
	log.Println("Listening on ", address)
	log.Fatal(http.ListenAndServe(address, nil))
}

File: helloserver.js
(function() {
  var http = require('http');
  
  var srv = http.createServer(function(req, res) {
    res.end('Hello, ' + req.url);
  });

  var port = 9992;
  console.log("Listening on port " + port);
  srv.listen(port, '127.0.0.1');
})();
